Runtime/Scripts/ControllerPointer.cs

```diff
@@ -11,72 +11,123 @@ namespace Cognitive3D
     [AddComponentMenu("Cognitive3D/Internal/Controller Pointer")]
     public class ControllerPointer : MonoBehaviour, IControllerPointer
     {
+        [HideInInspector]
         public Material DefaultPointerMat;
-        Material FocusPointerMat;
-        private bool focused;
-        public bool DisplayLineRenderer = true;
-        public LineRenderer LineRendererOverride;
-        Vector3[] pointsArray;
 
-        LineRenderer lr;
+        [HideInInspector]
```

Comment:
Updated script to easily configure more hand tracking in future. Pinch gesture, custom activation, etc.